web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -3169,6 +3171,15 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
                   </>
                 ),
               },
+              {
+                name: 'context.taskLockType',
+                type: 'boolean',
+                label: `Allow concurrent ${appendToExisting ? 'append' : 'replace'} tasks`,
+                defaultValue: undefined,
+                valueAdjustment: v => (v ? (appendToExisting ? 'APPEND' : 'REPLACE') : undefined),
+                adjustValue: v => v === (appendToExisting ? 'APPEND' : 'REPLACE'),
+                info: <p>Allows or forbids concurrent compactions.</p>,
```

Comment:
should this say concurrent append tasks instead of compactions since its for new ingestions? Similar comment on other mix of use of append tasks and concurrent compaction just to make sure everything is consistent

Also, i think this feature is experimental, should we label it as such?

Reply:
Yes, I agree